src/outagesim/backup_reliability.jl

```diff
@@ -434,20 +697,21 @@ Return a dictionary of inputs required for backup reliability calculations.
     -gen_operational_availability::Real = 0.9998        Fraction of year generators not down for maintenance
     -gen_failure_to_start::Real = 0.0066                Chance of generator starting given outage
     -gen_failure_to_run::Real = 0.00157                 Chance of generator failing in each hour of outage
-    -num_gens::Int = 1                                  Number of generators. Will be determined by code if set to 0 and gen capacity > 0.1
-    -gen_capacity::Real = 0.0                           Backup generator capacity. Will be determined by REopt optimization if set less than 0.1
+    -num_gen::Int = 1                                  Number of generators. Will be determined by code if set to 0 and gen capacity > 0.1
+    -gen_capacity_kw::Real = 0.0                           Backup generator capacity. Will be determined by REopt optimization if set less than 0.1
     -num_battery_bins::Int = 100                        Internal value for modeling battery
     -max_outage_duration::Int = 96                      Maximum outage hour modeled
     -microgrid_only::Bool = false                       Determines how generator, PV, and battery act during islanded mode
 ```
 """
-function backup_reliability_inputs(d::Dict, p::REoptInputs; r::Dict = Dict())::Dict
+function backup_reliability_inputs(;d::Dict, p::REoptInputs, r::Dict = Dict())::Dict
```

Comment:
This function should determine just the subset of ERP inputs that come from reopt inputs/results. It will be used to populate the ERP fields initially after a reopt run, at which point the user hasn't provided any ERP specific inputs yet.